packages/odata-common/src/de-serializers/payload-value-converter.ts

```diff
@@ -2,14 +2,6 @@
 
 import BigNumber from 'bignumber.js';
 
-/**
- * @internal
- * @deprecated I think this should be removed and replaced by the number functions below.
- */
-export function convertToNumber(value: any): number {
```

Comment:
Sorry I was not very clear with this. I was wondering whether we should use `deserializeToNumber` and `serializeFromNumber` instead of `convertToNumber`. I discussed this with @florian-richter and we didn't see an issue with this, but didn't want to put this change into our previous PR to separate concerns.